src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -384,15 +386,15 @@ public static DynamoModel Start(StartConfiguration configuration)
             return new DynamoModel(configuration);
         }
 
-        protected DynamoModel(StartConfiguration configuration)
+        protected DynamoModel(StartConfiguration config)
         {
             ClipBoard = new ObservableCollection<ModelBase>();
             MaxTesselationDivisions = 128;
```

Comment:
Sorry to clarify, the 128 should be a declared constant e.g. 
final int MAX_TESSELATIONS_DIVISIONS_DEFAULT = 128;